frontend/src/App.tsx

```diff
@@ -98,6 +99,7 @@ const router = createBrowserRouter(
                 <Route path='help' element={<HelpPage />} />
                 <Route path='tournaments'>
                     <Route index element={<TournamentsPage />} />
+                    <Route path='round-robin' element={<TournamentViewer />} />
```

Comment:
Why is this change here?

Reply:
feel free to remove this in the grad branch, I'm already working on this in the tournament UI